beluga_ros/include/beluga_ros/amcl.hpp

```diff
@@ -132,11 +132,37 @@ class Amcl {
       const AmclParams& params,
       execution_policy_variant execution_policy);
 
-  /// Returns a reference to the current set of particles.
+  /// Returns a reference to the current set of particles
   [[nodiscard]] const auto& particles() const { return particles_; }
 
   /// Returns a reference to the current likelihood field.
-  [[nodiscard]] const auto& get_likelihood_field() const { return likelihood_field_; }
+  [[nodiscard]] const auto& get_likelihood_field() const {
```

Comment:
@LaBruma nit: throughout the codebase, we usually use the noun without a leading `get_`. See `particles()` for instance. Consider this just being `likelihood_field()`. Same for likelihood_field_origin()`.